src/test/bip324_tests.cpp

```diff
@@ -11,8 +11,10 @@
 #include <util/strencodings.h>
 
 #include <array>
-#include <vector>
 #include <cstddef>
+#include <span.h>
+#include <stdint.h>
```

Comment:
Note that we keep our includes, and stardard library headers separate, so `span.h` should be added above. We also try use the cpp headers, [so cstdint over stdint.h](https://en.cppreference.com/w/cpp/header/cstdint) (regardless of what IWYU suggests).
```suggestion
#include <cstdint>
```